.github/workflows/quality-assurance.yml

```diff
@@ -32,32 +32,35 @@ jobs:
           cache-version: 4
         uses: actions/cache@c3f1317a9e7b1ef106c153ac8c0f00fed3ddbc0d # pin v3.0.4
         with:
-          key: ${{ runner.os }}-rust-${{ env.rust-version }}-cache-${{env.cache-version }}-${{ hashFiles('Cargo.lock') }}
+          key: ${{ runner.os }}-rust-${{ env.rust-version }}-cache-${{ env.cache-version }}-${{ hashFiles('Cargo.lock') }}
           restore-keys: |
-            ${{ runner.os }}-rust-${{ env.rust-version }}-cache-${{env.cache-version }}
+            ${{ runner.os }}-rust-${{ env.rust-version }}-cache-${{ env.cache-version }}
           path: |
             target
             ~/.cargo/bin
             ~/.cargo/registry/index
             ~/.cargo/registry/cache
             ~/.cargo/git/db
 
-      - name: Debug rust cache hit
-        run: echo ${{ steps.cache-rust.outputs.cache-hit }}
-
-      - if: steps.cache-rust.outputs.cache-hit != 'true'
-        uses: actions-rs/toolchain@b2417cde72dcf67f306c0ae8e0828a81bf0b189f # pin v1.0.6
+      - uses: actions-rs/toolchain@b2417cde72dcf67f306c0ae8e0828a81bf0b189f # pin v1.0.6
         with:
           toolchain: ${{ env.rust-version }}
           components: rustfmt, clippy
           default: true
+          override: true
```

Comment:
Override is used to bypass cache ? can you add a small comment about this

Reply:
not really, override

> Set installed toolchain as an override for the current directory
> https://github.com/actions-rs/toolchain#inputs